cli/src/pcluster/aws/aws_resources.py

```diff
@@ -48,6 +49,15 @@ def name(self):
         """Return the name of the stack."""
         return self._stack_data.get("StackName")
 
+    @property
+    def resources(self):
+        """Return the resources of the stack."""
+        if not self.__resources:
+            from pcluster.aws.aws_api import AWSApi  # noqa: F401 pylint: disable=import-outside-toplevel
```

Comment:
why do we need to import locally instead of top level?

Reply:
Importing on top leads to circular reference between Python files

Reply:
And this is a bad red flag for us.
I agree that we can consider this as out of scope for this specific PR, but we must address this issue in a follow up activity.